### DISCUSSION TITLE
How to fix logs not being able to be created?

### DISCUSSION THREAD

**Discussion by Michael21107** [scripthookvdotnet/scripthookvdotnet#1492] [Q&A] [answered]
I got this from another guy, neither SHV, SHVDN nor dinput8 makes logs for him (he got no other ''log making'' mods afaik) but he can open his SHVDN console and he saw this among other things
![image](https://github.com/user-attachments/assets/e5fa6797-b7b0-4441-8185-4baa462de0c9)

**Comment by kagikn** [comment, marked as answer]
he needs to give write or full permissions to files and folders (or at least the 2 log files) before SHV and SHVDN can write logs. A `System.UnauthorizedAccessException` is thrown when some C# program tries to read or write files but gets denied due to insufficient permissions.

**Reply by Michael21107** [reply]
works, thanks a lot mate
